plugins/show_contacts.py

```diff
@@ -5,14 +5,15 @@
 clustermols by Matthew Baumgartner.
 The advantage of this package is it requires many fewer dependencies.
 '''
+from __future__ import print_function
 
 import sys,os
-import tkSimpleDialog
-import tkMessageBox
-import tkFileDialog
+import tkinter.simpledialog
```

Comment:
This will only work in Python 3, not Python 2. I would typically suggest to follow https://pymolwiki.org/index.php/2to3#Imports - but in your case, you can simply remove all `tkinter` imports, they are not used in your script.